### ISSUE TITLE
Gerald ghosting, can't cut through material

### ISSUE THREAD

**Issue by MatthewCroughan** [DoESLiverpool/somebody-should#1031]
Same as usual, this is a very consistent problem, occurring at inconsistent times. Does anyone know why it would be more powerful as the etch goes on towards the bottom left? The bed is not that wonky, and the piece is only small. This is probably mirror alignment.

![img_20190118_085217](https://user-images.githubusercontent.com/26458780/51375929-0129a580-1aff-11e9-8387-926cc27af813.jpg)

**Comment by Sean-anotherone** [comment]
Alignment (mostly) and bed level are probably both contributors to that.

The beam comes from the primary mirror down the left side of the machine where it hits the secondary, which then sends it to the z mirror on the cutting head.
Alignment will always be easiest to maintain in the top left corner, and hardest in the bottom right corner.
An overhaul is the proper fix but some time spent on realignment will get it working for a while (again).

Now it has been running for a while, the new tube probably needs readjustment to ensure it's hitting the primary mirror square and level.

**Comment by dazz-ling** [comment]
I did a quick alignment on Saturday which seemed to sort it out as far a cutting goes. Managed to cut a 1m x 600mm, 6mm plywood sign with no issues. 
Pre testing did show some ghosting from the right side of the bed . 
I didn't do any complex etching so this may still need testing and a full tune up sounds a good idea

**Comment by MatthewCroughan** [comment]
@DoESLiverpool/laser-maintenance 
![img_20190121_031652](https://user-images.githubusercontent.com/26458780/51458065-ca949a80-1d4b-11e9-8759-2a396aaa73be.jpg)

Similar results after testing again today, although somewhat better. These are the same files I've used fot months, so they are consistent.

**Comment by KarlDunkerley** [comment]
If anyone wants some help tackling this then give me a shout as I have a fairly free week for once.
Maybe we should get the laser crew together again asap.
Cheers, 

Karl Dunkerley

Mob 07710 353 122

eml karldunkerley@yahoo.com 

    On Monday, 21 January 2019, 07:12:21 GMT, MatthewCroughan <notifications@github.com> wrote:  
 
 
@DoESLiverpool/laser-maintenance


Similar results after testing again today, although somewhat better. These are the same files I've used fot months, so they are consistent.

**Comment by magman2112** [comment]
With the laser move and some realignment that @Sean-anotherone and I did on Saturday and Sunday, Gerald now appears to be cutting a lot better. 

A faulty mirror was also replaced and all of the other mirrors were removed and thoroughly cleaned which will have significantly affected the laser power. The laser tube was also levelled and the vertical alignment through the cutting head was also adjusted.

Closing this issue now as the primary causes for this issue have now been addressed.
